### ISSUE TITLE
Missing Files?

### ISSUE THREAD

**Issue by StephenClement** [ducksboard/gridster.js#404]
I recently stumbled upon gridster when I was looking for a metro style tile setup for a new project and I was very impressed with what I saw on the website. Unfortunately when I came to download the package off of github it seemed to be for some reason incomplete. I tried to use the test file in the test folder and all I saw was a header with about 10 bullet points from the list in the test file. After further inspection of the test file code I noticed that there were references to a libs folder which is absent from the current build. Am I missing something? Thanks!

**Comment by StephenClement** [comment]
Something is seriously wrong with this release. I don't know what happened, but the files are not working for me at all. In order to even get a functional demo for my project I had to go and clone all of the files off of gridster.net. Can someone possibly restore this back to a more stable release?

**Comment by GioValentin** [comment]
Yea i am having the same problem and not getting any feedback what so ever. Did you find a solution?

**Comment by StephenClement** [comment]
Yeah, I found that the issue I was having was with the test file. The test file doesn't function properly, but I believe that is because it is unmaintained and isn't necessarily designed to work properly. The actual gridster file functions fine, but you are going to have to develop your own page that implements it.

**Comment by GioValentin** [comment]
Yea thats exactly what i ended up doing. But thank you for your feedback.

**Comment by ij0n** [comment]
I have the same issue here. Would be nice, if the test.html could be fixed by someone, so beginners get a smoother start.
